ingestion/src/metadata/ingestion/source/metadata/amundsen.py

```diff
@@ -195,21 +202,44 @@ def create_tags(self, tags):
     def create_table_entity(self, table):
         try:
             service_name = table["database"]
-            service_type = self.database_service_map.get(
-                service_name.lower(), DatabaseServiceType.Mysql.value
-            )
-
             # TODO: use metadata.get_service_or_create
             service_entity = self.get_database_service(service_name)
-            database = Database(
-                id=uuid.uuid4(),
+            database_request = CreateDatabaseRequest(
                 name="default",
-                service=EntityReference(id=service_entity.id, type=service_type),
+                description=table.get("description"),
+                service=EntityReference(id=service_entity.id, type="databaseService"),
             )
-            database_schema = DatabaseSchema(
+
+            yield database_request
+
+            database_fqn = fqn.build(
+                self.metadata,
+                entity_type=Database,
+                service_name=service_name,
+                database_name=database_request.name.__root__,
+            )
+
+            database_object = self.metadata.get_by_name(
+                entity=Database, fqn=database_fqn
+            )
+
+            database_schema_request = CreateDatabaseSchemaRequest(
                 name=table["schema"],
-                service=EntityReference(id=service_entity.id, type=service_type),
-                database=EntityReference(id=database.id.__root__, type="database"),
+                description=database_request.description.__root__,
```

Comment:
same goes here we don't have to put database description as databaseSchema description

Reply:
Ok.